src/ui/components/shared/JumpToCodeButton.tsx

```diff
@@ -27,13 +28,14 @@ export function JumpToCodeButton({
   onClick,
   status,
   targetExecutionPoint,
+  className = "",
 }: JumpToCodeButtonProps) {
   const [isHovered, setIsHovered] = useState(false);
 
   const jumpToCodeButtonAvailable = status === "not_checked" || status === "found";
 
   const jumpToCodeButtonClassname = classnames(
-    "transition-width flex items-center justify-center rounded-full  duration-100 ease-out h-6",
+    `jump-to-code transition-width flex items-center justify-center rounded-full  duration-100 ease-out h-6 ${className}`,
```

Comment:
What is the `jump-to-code` class name supposed to accomplish?

Reply:
Oh. I see. That's a nested selector style.

I think we should avoid those. Can we do that in a different way here @jasonLaster?

The way we normally do this is to pass in class names that point to CSS module styles.